### PR TITLE
nsync: fix clang build

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-nsync/PKGBUILD [hunk 1]
```diff
@@ -30,41 +30,36 @@ apply_patch_with_msg() {
...
     -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
     -DCMAKE_BUILD_TYPE=Release \
-    -DNSYNC_ENABLE_TESTS=OFF \
+    -DNSYNC_ENABLE_TESTS=ON \
     -DBUILD_SHARED_LIBS=ON \
-    ../${_archive}
+    ../${_realname}-${pkgver}
 
-  make -j1
+  make
```
Comment (neheb) [inline mingw-w64-nsync/PKGBUILD:52]: cmake --build is better

File: mingw-w64-nsync/PKGBUILD [hunk 2]
```diff
@@ -30,41 +30,36 @@ apply_patch_with_msg() {
...
 build() {
-  [[ -d "${srcdir}/build-${MINGW_CHOST}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}"
-  mkdir -p "${srcdir}/build-${MINGW_CHOST}"
-  cd "${srcdir}/build-${MINGW_CHOST}"
+  [[ -d "${srcdir}/build-${MSYSTEM}" ]] && rm -rf "${srcdir}/build-${MSYSTEM}"
+  mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"
 
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
   ${MINGW_PREFIX}/bin/cmake \
     -G "MSYS Makefiles" \
```
Comment (neheb) [inline mingw-w64-nsync/PKGBUILD:45]: Ninja is better

Comment (raedrizqie) [reply mingw-w64-nsync/PKGBUILD]: ok, done

**Comment by raedrizqie** [comment]
thanks!

**Pull request merged**